tests/unit/UserAvatarUtilsTest.ts

```diff
@@ -0,0 +1,275 @@
+import * as defaultAvatars from '@components/Icon/DefaultAvatars';
```

Comment:
given this is not fixing anything, is it better to push the passing tests in one PR, and then push this PR to prove it doesn't change the logic?